src/modules/governance/proposal/VoteInfo.tsx

```diff
@@ -42,9 +47,45 @@ export function VoteInfo({ proposal }: VoteInfoProps) {
 
   return (
     <Paper sx={{ px: 6, py: 4, mb: 2.5 }}>
-      <Typography variant="h3" sx={{ mb: 8 }}>
-        <Trans>Your voting info</Trans>
-      </Typography>
+      <Row
+        sx={{ mb: 8 }}
+        caption={
+          <>
+            <Typography variant="h3">
+              <Trans>Your voting info</Trans>
+            </Typography>
+            {network && (
+              <Box
+                sx={{
+                  display: 'flex',
+                  alignItems: 'center',
+                  color: 'text.secondary',
+                }}
+              >
+                <Typography variant="caption">
+                  <Trans>This vote is free on</Trans>
```

Comment:
We currently are not using the relayer for gasless voting. Maybe for now we could just change this text to `Voting is on` and then we can update later.

Reply:
@MareskoY can you update we can get this into main